app/views/govuk_publishing_components/components/_tabs.html.erb

```diff
@@ -0,0 +1,33 @@
+<%
+  tabs ||= []
+%>
+<% if tabs.count > 1 %>
+  <div class="govuk-tabs gem-c-tabs" data-module="tabs">
+    <h2 class="govuk-tabs__title">
+      Contents
+    </h2>
+    <ul class="govuk-tabs__list">
+      <% tabs.each do |tab| %>
+      <li class="govuk-tabs__list-item">
+        <a class="govuk-tabs__tab" href="#<%= tab[:id] %>">
+            <%= tab[:title] %>
+          </a>
+      </li>
+      <% end %>
+    </ul>
+    <% tabs.each do |tab| %>
+      <section class="govuk-tabs__panel" id="<%= tab[:id] %>">
+        <h2 class="govuk-heading-l"><%= tab[:title] %></h2>
+          <% if tab[:content].html_safe? %>
```

Comment:
I'm not sure we need this - if `tab[:content]` is not `html_safe`, it will be automatically escaped.

Reply:
Fixed.